servicecontrol/servicecontrol-in-practice.md

```diff
@@ -0,0 +1,27 @@
+title: ServiceControl in Practice
+summary: Details how to think about the use of ServiceControl in production environments
+tags:
+- ServiceControl
+---
+
+ServiceControl is a tool that can accommodate many situations. Certain features come with a cost and in some cases the cost of having a feature active is not worth the performance impact on the system.
+
+For example, in your development environment you will want a lot of logging information to support problem analysis. For many users the [Debug Session](/servicecontrol/plugins/debug-session.md) plugin is really useful at this stage of the application life-cycle but you would never put that plugin into production. For the same reason, we advise that customers never use the [Saga Audit](/servicecontrol/plugins/saga-audit.md) plugin outside of development.
+
+Making decisions about the use of the other plugins and features requires a little more thought to balance the smooth running of a system with actual business requirements. The temptation to 'just use them all' as a catch-all insurance policy is not a good choice. The cost of having some of these features in play can actually cause issues later on if you are not prepared.
+
+[Auditing](/nservicebus/operations/auditing.md) is a good example of how the cost of having a record of everything may cause significant impact on the system. It might be worth considering turning audit off in some or all endpoints in production. Or if you absolutely need it move the auditing to another place like your Business Intelligence database or somewhere that has better resources dedicated to the problem. Remember audits in ServiceControl are primarily for performing system analysis using ServiceInsight. If you are not using ServiceInsight in production then do not use audits in production without having good reason.
```

Comment:
The cost is turning on the feature. The business value is having a record of everything. 

Rather than jumping to the solution "turn off audits", I'd start with questioning the value of audits for certain types of endpoints. Then, if the value of audits is low, consider turning them off. 

The point about ServiceInsight is a good one. If you aren't looking, why do it.